src/utils/test/allIconInfo.test.ts

```diff
@@ -0,0 +1,28 @@
+import type { SimpleIcon } from 'simple-icons';
+
+import customIcons from '../../constants/custom-icons.json';
+import { makeIconInfoArray } from '../allIconInfo';
+
+describe('check custom icons', () => {
+  let allIconArr: SimpleIcon[];
+
+  beforeAll(() => {
+    allIconArr = makeIconInfoArray();
+  });
+
+  it('check last added icon title', () => {
+    const lastCustomIcon = allIconArr.at(-1);
```

Comment:
good to see `at(-1)`

Reply:
In the allIconInfo.ts file, customIcon is always inserted after simple-icons, so you can write a test that compares the title of the last icon in the customIcon array with the title of the last icon in allIconArr?? cool!